apps/web/src/features/organizations/components/MembersList/RemoveMemberModal.tsx

```diff
@@ -9,10 +9,12 @@ const RemoveMemberDialog = ({
   userId,
   memberName,
   handleClose,
+  isInvite = false,
```

Comment:
I think we also need to call `deleteInvite` instead of `deleteMember` when this flag is present right?